README.md

```diff
@@ -46,6 +46,10 @@ git clone --recursive git@github.com:openmultiplayer/open.mp
 
 Note the use of the `--recursive` argument, because this repository contains submodules.
 
+If you don't use this parameter, you will need to download the files manually and place them in their respective folders, or you will encounter errors during the build process.  
```

Comment:
this **git** repository contains **git** submodules
if they don't know how to use git then it's not really our problem, we shouldn't even mention this *method* of retrieving submodules, because then it's not going to be the right revision

Reply:
> this **git** repository contains **git** submodules if they don't know how to use git then it's not really our problem, we shouldn't even mention this _method_ of retrieving submodules, because then it's not going to be the right revision

It is not about know how to use git, the command above shows how to download with sub modules...
I just wanted to make life easier to people who don't like to use this parameter and want to download manually, but you are right, if dependencies change we will need to change on README.md too, so i'll remove this part...